### PR TITLE
CDAP-7458 Minor fixes

### PR DESCRIPTION
**Pull request by awholegunch**
Minor fixes to the doc changes from CDAP-7458.

### CODE REVIEW

File: cdap-docs/admin-manual/source/operations/tx-maintenance.rst [hunk 1]
```diff
@@ -35,11 +35,11 @@ To prune the invalid list manually, follow these steps:
...
 
 - Find the minimum prune time across all queues by running the tool ``SimpleHBaseQueueDebugger``.
-  Note that authorization is disabled when running the SimpleHBaseQueueDebugger, so that cdap can read all
-  users' tables.
-  This should print lines such as::
+  Note that authorization is disabled when running the ``SimpleHBaseQueueDebugger`` so that CDAP can read all
+  users' tables. This should print lines such as::
 
     $ /opt/cdap/master/bin/svc-master run co.cask.cdap.data.tools.SimpleHBaseQueueDebugger
+
```
Comment (anwar6953) [inline cdap-docs/admin-manual/source/operations/tx-maintenance.rst:41]: there's actually a space?

Comment (awholegunch) [reply cdap-docs/admin-manual/source/operations/tx-maintenance.rst:41]: No, but I wanted to make it easier for users to read the command, separate from what is returned. It was all running together...do you see that as a problem?

Comment (anwar6953) [reply cdap-docs/admin-manual/source/operations/tx-maintenance.rst:41]: Well, maybe its just a nitpick, but then it deviates from how the lines actually look on the user's terminal, which we imply its the same.

Comment (awholegunch) [reply cdap-docs/admin-manual/source/operations/tx-maintenance.rst:41]: I will revise it then to make it clearer.

**Comment by anwar6953** [comment]
lgtm

**Pull request merged**